hash_practice/exercises.py

```diff
@@ -2,18 +2,46 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n)
+        Space Complexity: O(n)

```

Comment:
👍 The time complexity is right if the words are limited in length, if they could be any length, the time complexity is O(n * m log m) because each word is sorted.